### DISCUSSION TITLE
Weekly Update 2020-11-23

### DISCUSSION THREAD

**Discussion by frol** [near/near-explorer#487] [Show and tell]
Our previous update is here: #481

Find our Objectives and Key Results for 2020 Q4: #467

@icerove @khorolets @corwinharrell Please, bring us your updates and let me know if you are blocked at something

# Summary by Focus

## Explorer for Non-tech Users

No updates this week

## Explorer for Contract Developers

* Indexer for Explorer is getting more stable, and we believe it is the last full-resync we pushed last week. Thus, we are almost there in regards to the data source which Explorer can use instead of its own incomplete indexer. See more details from @khorolets below
* Explorer migration to the Indexer for Explorer database required quite a massive effort, and thanks @icerove for #477 and #478 which bring us close to being able to use Indexer for Explorer as if nothing has changed. This step will unblock us from iterating on more features providing mode details about transactions (including cross-contract calls and their results).

## Explorer for Validators

No updates this week

**Comment by khorolets** [comment]
Previous week I've planned to deal with missing data issue where we found out that we don't miss any data, the Indexer for Explorer is running too fast and we're experiencing retry collisions. We've tried to run Indexer on a more powerful machine and the effect of "missing data" has decreased. So we decided to increase delay between attempts (corresponding PR is here https://github.com/near/near-indexer-for-explorer/pull/53)

Also I've updated NEAR Indexer Framework, moved ExecutionOutcomes from separate field into chunk as we now preserve the order of outcomes. Indexer for Explorer also has been updated and currently running for mainnet from genesis.

StateChanges https://github.com/near/near-indexer-for-explorer/issues/9 are planned for this week along with updating all the README and other info to release Indexer for Explorer 0.2.0.
